Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Shortcut+Default.swift
+//  Easydict
+//
+//  Created by Sharker on 2024/2/5.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Defaults
+import Magnet
+
+extension Shortcut {
+    // set defalut for app shortcut
+    func setDefaultForShortcut() {
+        setDefaultForGlobalShortcut()
+        setDefaultForAppShortcut()
+    }
+
+    private func setDefaultForGlobalShortcut() {
+        Defaults[.selectionShortcut] = KeyCombo(key: .d, cocoaModifiers: .option)
+        Defaults[.snipShortcut] = KeyCombo(key: .s, cocoaModifiers: .option)
+        Defaults[.inputShortcut] = KeyCombo(key: .a, cocoaModifiers: .option)
+        Defaults[.screenshotOCRShortcut] = KeyCombo(key: .f, cocoaModifiers: [.option, .shift])
+        Defaults[.showMiniWindowShortcut] = KeyCombo(key: .s, cocoaModifiers: .option)
```

Comment:
Wrong default values, it's just `option + a, s, d, f` one-line key order.

<img width="234" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/2119a489-7c03-4958-96fa-17256421aec7">